vizro-core/docs/pages/user-guides/card-button.md

```diff
@@ -404,12 +405,12 @@ and provide an attribute selector to select images with that matching URL hash.
     [guide on navigation](navigation.md).
 
 
-A navigation card allows you to navigate to a different page via a click on the card area.
+A navigation card enables you to navigate to a different page via a click on the card area.
 To create a navigation card, do the following:
 
 - Insert the [`Card`][vizro.models.Card] into the `components` argument of the [`Page`][vizro.models.Page]
-- Provide the `text` parameter with a title and some description
-- Provide the `href` parameter (relative or absolute URL)
+- Give the `text` parameter with a title and some description
+- Give the `href` parameter (relative or absolute URL)
```

Comment:
**Provide** sounds better here - doesn't it?

"Give the `text` parameter with a title and some description" sounds a bit odd to me 😄